playbooks/roles/edxapp/files/microsite_theming.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All Rights Reserved.
+
+# Remove if edx-microsite folder exists
+if [[ -d /edx/app/edxapp/edx-microsite ]]; then
+  echo "*** Deleting the existing edx-microsite/ folder ***"
+  sudo rm -fr /edx/app/edxapp/edx-microsite/
+fi
+
+cd /edx/app/edxapp/
+
+# Download microsite assets from github to folder ../edx-microsite 
+sudo git clone https://github.com/ms-manikarthik/edx-microsite.git
```

Comment:
don't merge until you've moved this to a permanent /Microsoft/ repo and updated this url accordingly.